src/app/components/rich-text-editor/prosemirror-styles.scss

```diff
@@ -4,6 +4,11 @@
   line-height: initial;
   color: $text-primary;
 
+  .selection-marker {
+    background: #338FFF;
+    color: white;
```

Comment:
Im my case, the default highlight is `color: black` hence it changes to white when I focus the link editing control. As such, I think this should be black so that it doesn't change and looks more seamless